docs/CSBR.md

```diff
@@ -858,22 +859,20 @@ Private Keys corresponding to CA Keys MUST be stored in accordance with BR Secti
 
 #### 6.2.7.2 Private key storage for Timestamp Authorities
 
-A Timestamp Authority MUST protect its signing key using a process that is at least to FIPS 140-2 Level 3, Common Criteria EAL 4+ (ALC\_FLR.2), or higher. The CA MUST protect its signing operations in accordance with the CA/Browser Forum's Network Security Guidelines.
+A Timestamp Authority MUST protect its signing key using a process that is at least to FIPS 140-2 Level 3, Common Criteria EAL 4+ (ALC\_FLR.2), or higher. The Timestamp Authority MUST protect its signing operations in accordance with the CA/Browser Forum's Network Security Guidelines.
 
 #### 6.2.7.3 Private key storage for Signing Services
 
-The Signing Service MUST ensure that a Subscriber's Private Key is generated, stored, and used in a secure environment that has controls to prevent theft or misuse. A Signing Service MUST enforce multi-factor authentication to access and authorize Code Signing and obtain a representation from the Subscriber that they will securely store the tokens required for multi-factor access. A system used to host a Signing Service MUST NOT be used for web browsing. The Signing Service MUST run a regularly updated antivirus solution to scan the service for possible virus infection. The Signing Service MUST comply with the Network Security Guidelines as a "Delegated Third Party".
+The Signing Service MUST ensure that a Subscriber's Private Key is generated, stored, and used in a secure environment that has controls to prevent theft or misuse. A Signing Service MUST enforce multi-factor authentication or secure server-to-server authentication to access and authorize the signing of Application Code. The Signing Service MUST comply with the Network Security Guidelines.

```

Comment:
How is "secure server-to-server authentication" defined? We should either provide some illustrative examples so that those implementing and auditing this requirement have some approximation of what we mean by "secure", or remove the word "secure".

Also, "Application Code" is not defined. Suggest removing the word "Application".

We repeat that the Signing Service MUST comply with NetSec. See my previous comment about possibly updating section 6.7 so we don't have to repeat this requirement multiple times in the CSBRs.

Reply:
I added in "secure server-to-server authentication" to support automated signing, which might be the largest use case. I also have not had anyone state what term or methods we should use for machine-to-machine authentication. However, I believe that CAs have already implemented this type of process for automated certificate issuance. I would prefer we make a statement which gives an alternative to "multi-factor".

Reply:
We probably need more details from CAs that have implemented this type of process so the replacement language is equally secure (or more secure) than the previous one. The way this proposal is written, without any description or controls for the server-to-server authentication, may weaken the process. I mean, on one hand we require multi-factor authentication and on the other (server-to-server) it could be just a plain password which never expires.

Reply:
I agree, we need more details on minimum requirements for secure server to server authentication.  Let me ask our security team what we can recommend here.